sdk/keyvault/tests.yml

```diff
@@ -12,7 +12,7 @@ extends:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
       Canary:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-        Location: 'eastus2euap'
+        Location: 'centraluseuap'
```

Comment:
Do we have to be in canary? Was this mostly to test early? Canary tends to always be less stable than the production environment so was curious.  Also is this a .NET decision or something that we do across? I see different SDKs doing it differently.

Reply:
Canary was an intentional decision, yes. We provide backstop testing so it helps spot any regressions early.

Reply:
got it, thanks.